crates/sargon/src/profile/logic/profile_network/profile_network_details.rs

```diff
@@ -118,12 +118,7 @@ impl AuthorizedPersonaSimple {
 
         let shared_accounts = self.accounts_for_display(non_hidden_accounts)?;
 
-        let has_auth_signing_key = match &persona.security_state {
-            EntitySecurityState::Unsecured { value: uec } => {
-                uec.authentication_signing.is_some()
-            }
-            _ => false, // TODO change that
-        };
+        let has_auth_signing_key = persona.is_securified();
```

Comment:
`AuthorizedPersonaDetailed` struct contains that field. Not sure we use it anywhere in hosts, but I did not wanna change that. And the assignment ` let has_auth_signing_key = persona.is_securified();` is correct. ALL Securified Entties has an auth signing factor. No Unsecurified has one.

Reply:
@CyonAlexRDX We discussed using `transaction_signing` factor instance when an entity is not securified. Doesn't that mean that `has_auth_signing_key` will always be true? Or does `has_auth_signing_key` mean like a true ROLA key?

Reply:
Rather `has_auth_signing_key` will always be `false` for unsecurified entities. But unsecurified entities can ofc proof-ownership (which it does using `transaction_signing`.)

I'm not sure this field is at all relevant. I think it might have been used by iOS behind `DEBUG` flag. So we should not put too much importance on this field.

Reply:
Thanks Alex, indeed it is used to disable the button that created the rola key for unsecurified entities. Indeed unneeded.